### PR TITLE
refactor(VoiceRegion): removed `vip` property

### PR DESCRIPTION
**Pull request by almeidx**
## About
- https://github.com/discord/discord-api-docs/pull/3908

## Status

- [x] These changes have been tested against Discord API or do not contain API change.
- [ ] This PR includes only documentation changes, no code change.
- [x] This PR introduces some Breaking changes.

### CODE REVIEW

**Review by Helloyunho — APPROVED** [review]

File: src/types/voice.ts [hunk 1]
```diff
@@ -50,8 +50,6 @@ export interface VoiceRegion {
   id: string
   /** Name of the region */
   name: string
-  /** True if this is a vip-only server */
-  vip: boolean
```
Comment (Helloyunho) [inline src/types/voice.ts:54]: actually set it as `false` instead of deleting it so it won't break the api

Comment (MierenManz) [reply src/types/voice.ts:54]: Also handy to add @deprecate to it 😉

Comment (almeidx) [reply src/types/voice.ts:54]: I might have deleted my fork and now have to remake the pull request

**Comment by almeidx** [comment]
Superseded by #254

**Pull request closed without merging**